docs/hub/spaces-dependencies.md

```diff
@@ -18,6 +18,6 @@ The default Spaces environment comes with several pre-installed dependencies:
 
 ## Adding your own dependencies
 
-If you need other Python packages to run your app, add them to a **requirements.txt** file at the root of the repository. The Spaces runtime engine will create a custom environment on-the-fly.
+If you need other Python packages to run your app, add them to a **requirements.txt** file at the root of the repository. The Spaces runtime engine will create a custom environment on-the-fly (you can also add a **pre-requirements.txt** file describing dependencies that will be installed before your main dependencies. It can be usefull if you need to update pip itself).
```

Comment:
```suggestion
If you need other Python packages to run your app, add them to a **requirements.txt** file at the root of the repository. The Spaces runtime engine will create a custom environment on-the-fly. You can also add a **pre-requirements.txt** file describing dependencies that will be installed before your main dependencies. It can be useful if you need to update pip itself.
```